### PR TITLE
Remove move "enabled" from Reoseta::Config to Rosetta class

### LINKED ISSUE #29: Enabled/disabled should be in Request.store
Otherwise, by using a class instance variable we may pollute other requests.

### PR DESCRIPTION
**Pull request by artero**
- Remove move "enabled" from Reoseta::Config to Rosetta class, close #29 
- Add Rosetta namespace to the Phrases class, cose #27

### CODE REVIEW

File: lib/rosetta.rb [hunk 1]
```diff
@@ -30,13 +29,39 @@ def add_phrase(**args)
...
+    end
+
+    def disable
+      self.active = false
+    end
+
+    private
+
+    def active
+      request[:actived].nil? ? false : request[:actived]
```
Comment (oriolbcn) [inline lib/rosetta.rb]: type in `actived`

File: lib/rosetta.rb [hunk 2]
```diff
@@ -30,13 +29,39 @@ def add_phrase(**args)
...
+    end
+
+    def disable
+      self.actived = false
+    end
+
+    private
+
+    def actived
+      request[:actived].nil? ? false : request[:actived]
```
Comment (oriolbcn) [inline lib/rosetta.rb]: typo in `actived`

File: lib/rosetta/repositories/local.rb [hunk 3]
```diff
@@ -2,20 +2,20 @@ module Rosetta
...
         'Repository Local'
       end
 
-      def self.build_link(keys, **_args)
+      def build_link(keys, **_args)
         path = I18n.load_path.flatten.find { |filename| file_has_phrase?(filename, keys) }
         path && Pathname.new(path).relative_path_from(Rails.root)
       end
 
-      def self.file_has_phrase?(filename, keys)
```
Comment (oriolbcn) [inline lib/rosetta/repositories/local.rb]: we can make this method and all the methods below `private` now

File: lib/rosetta/config.rb [hunk 4]
```diff
@@ -7,46 +7,13 @@ def initialize
       set_defaults!
     end
 
-    def onesky
-      @onesky ||= OneSky.new
-    end
-
     def set_defaults!
       @repository = Rosetta::Repositories::Local
```
Comment (oriolbcn) [inline lib/rosetta/config.rb]: We need to change it to call new on the repository class

File: lib/rosetta.rb (hunk 2 above)
Comment (artero) [reply lib/rosetta.rb]: Done

File: lib/rosetta/repositories/local.rb (hunk 3 above)
Comment (artero) [reply lib/rosetta/repositories/local.rb]: Done

**Pull request merged**